buildscripts/buildRelease.sh

```diff
@@ -130,7 +132,9 @@ if [ "$RunTests" == "true" ]; then
 	while read line ; do
 		echo "<p><span class=rvts6>$line</span></p>" >> UXAspectsTestsResults.html
 	done < UnitTestResults.txt
+	sed -i 's/\[1m//g' UXAspectsTestsResults.html
 	sed -i 's/\[4m//g' UXAspectsTestsResults.html
+	sed -i 's/\[22m//g' UXAspectsTestsResults.html
 	sed -i 's/\[24m//g' UXAspectsTestsResults.html
 	sed -i 's/\[31m//g' UXAspectsTestsResults.html
 	sed -i 's/\[32m//g' UXAspectsTestsResults.html
```

Comment:
I would suggest that just the summary of the jasmine run be presented here rather than the full output. This should be easier to read since it only presents the number of tests executed and the failures. It also does not include control characters. This can be accomplished by adding `display: 'none'` and `summary: true` to the options in grunt/jasmine.js.

I also suggest that only the output from the line "Testing Jasmine specs via PhantomJS" onwards be included in the HTML.